pconduit/doc.go

```diff
@@ -16,4 +16,4 @@
 Package conduit contains definitions of Conduit services available to connectors,
 as well as client that enable the usage of those.
```

Comment:
This is not really true, also the package is `pconduit`. Do we even need this comment / file? 😬

Reply:
@lovromazgon i don't think we do.